ChangeLog

```diff
@@ -14,6 +24,7 @@
 	* man/BermudanSwaption.Rmd: Modified to reflect DiscountCurve
 	option and example updated
 	
+>>>>>>> eddelbuettel/master
```

Comment:
From my development machine I only committed 2 files.

I think the issues derived from syncing your master with my fork on Github. I was concerned that the master was 7 commits ahead, so I used Github desktop to sync and followed the app’s directions to execute this. I then pulled to my machine. When I committed the changes I sent only 2 files, but the pull took the other files from the sync’d master. I had no intention of resubmitting the other files.


> On May 13, 2016, at 6:17 AM, Dirk Eddelbuettel <notifications@github.com> wrote:
> 
> In ChangeLog <https://github.com/eddelbuettel/rquantlib/pull/47#discussion_r63161899>:
> 
> > @@ -14,6 +24,7 @@
> >  	* man/BermudanSwaption.Rmd: Modified to reflect DiscountCurve
> >  	option and example updated
> >  	
> > +>>>>>>> eddelbuettel/master
> See above.
> 
> You still don't know how to update your repo correctly to not have merge conflicts. That is a little worrysome.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/eddelbuettel/rquantlib/pull/47/files/8704b4c2e29fe228f1d6f2d3e846aa6311f01c4b#r63161899>